### DISCUSSION TITLE
One click leave friend chat in Chambers of Xeric

### DISCUSSION THREAD

**Discussion by Mckegg** [runelite/runelite#15290] [Ideas]
Currently when you want to leave or reset a raid you have to click "Leave" in the friend chat menu, and another "Leave" menu option will pop up to confirm you want to leave. It would be nice to have the 2nd leave option removed to help reset raids faster

**Comment by Morgan-Demon** [comment]
This is already a feature in **Cox Additions** made by **Buchus**

**Reply by Mckegg** [reply]
Thank you, wasnt aware of this
